lib/oracle.js

```diff
@@ -529,6 +588,6 @@ Oracle.prototype.ping = function (cb) {
   this.execute('select count(*) as result from user_tables', [], cb);
 };
 
-require('./migration')(Oracle);
-require('./discovery')(Oracle);
-require('./transaction')(Oracle);
+require('./migration')(Oracle, oracle);
+require('./discovery')(Oracle, oracle);
```

Comment:
The migration/discovery mixin requires an additional arg - oracle (the instance of the driver).